### ISSUE TITLE
Travis job for publishing releases to GCS is broken

### ISSUE THREAD

**Issue by simonpasquier** [google/go-jsonnet#294]
IIUC Travis CI is supposed to push the release artifacts to GCS but all jobs have [failed](https://travis-ci.org/google/go-jsonnet/branches) from `v0.11.2` to `v0.13.0`.

```
$ gsutil ls -r gs://jsonnet/
gs://jsonnet/latest
gs://jsonnet/v0.10.0/:

gs://jsonnet/v0.10.0/darwin/:

gs://jsonnet/v0.10.0/darwin/amd64/:
gs://jsonnet/v0.10.0/darwin/amd64/jsonnet

gs://jsonnet/v0.10.0/linux/:

gs://jsonnet/v0.10.0/linux/amd64/:
gs://jsonnet/v0.10.0/linux/amd64/jsonnet
```

Looking at the Travis [logs](https://travis-ci.org/google/go-jsonnet/jobs/540879661#L551), it appears that some environment [variables](https://github.com/google/go-jsonnet/blob/master/release.sh#L6) aren't defined in Travis.

Related to https://github.com/google/go-jsonnet/issues/273#issuecomment-490147643.

**Comment by sbarzowski** [comment]
TBH I'm not sure what it was about, but I don't think it's relevant to the current version.

**Comment by simonpasquier** [comment]
It was about `go-jsonnet` binaries not being publicly available since v0.10.0.
The issue got superseded by #366 that you've just created.
